roles/pulp_redis/vars/Fedora-33.yml

```diff
@@ -0,0 +1,3 @@
+---
+_pulp_redis_server_name: redis
+_pulp_redis_conf_file: '/etc/redis/redis.conf'
```

Comment:
Is it needed that this file is split for different versions of fedora?

Reply:
I believe so as on previous versions of fedora config file is one level higher. (not in `redis` directory)